README.md

```diff
@@ -12,7 +12,39 @@ Please tag **@edx/educator-neem** on any PRs or issues.
 
 The front-end of our editable Gradebook feature.
 
-## Usage
+### What does this offer over the stock LMS Gradebook?
```

Comment:
We prefer "legacy" to describe the older implementation (throughout).

Reply:
Changed.